<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/app.ico</normaloff>:/app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionOtherSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionRedo"/>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
    <addaction name="actionStatusBar"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>运行(R)</string>
    </property>
    <addaction name="actionCompile"/>
    <addaction name="actionRun"/>
    <addaction name="actionCompileRun"/>
   </widget>
   <widget class="QMenu" name="menuSet">
    <property name="title">
     <string>设置(T)</string>
    </property>
    <addaction name="actionSet"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>搜索(S)</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSearch"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
   <addaction name="menuRun"/>
   <addaction name="menuSet"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionOtherSave"/>
   <addaction name="separator"/>
   <addaction name="actionRedo"/>
   <addaction name="actionUndo"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionCompileRun"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="toolTip">
    <string>gdf g</string>
   </property>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="actionOtherSave">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/save_as.png</normaloff>:/images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="actionPageSetting">
   <property name="text">
    <string>页面设置(&amp;U)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/edit_redo.png</normaloff>:/images/edit_redo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/edit_undo.png</normaloff>:/images/edit_undo.png</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除(&amp;L)</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
  </action>
  <action name="actionFindNext">
   <property name="text">
    <string>查找下一个(&amp;N)</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
  </action>
  <action name="actionGoTO">
   <property name="text">
    <string>转到(&amp;G)</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="actionTimeOrDate">
   <property name="text">
    <string>时间/日期(&amp;D)</string>
   </property>
  </action>
  <action name="actionAutoLine">
   <property name="icon">
    <iconset>
     <normaloff>:/iamges/auto_line.png</normaloff>
     <normalon>:/auto_line.png</normalon>:/iamges/auto_line.png</iconset>
   </property>
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/font.png</normaloff>:/images/font.png</iconset>
   </property>
   <property name="text">
    <string>字体(&amp;F)</string>
   </property>
  </action>
  <action name="actionStatusBar">
   <property name="icon">
    <iconset>
     <normaloff>:/iamges/auto_line.png</normaloff>
     <normalon>:/auto_line.png</normalon>:/iamges/auto_line.png</iconset>
   </property>
   <property name="text">
    <string>
状态栏(&amp;S)</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于记事本(&amp;A)</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>运行(R)</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="text">
    <string>编译(C) </string>
   </property>
  </action>
  <action name="actionCompileRun">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/compilerun.png</normaloff>:/images/compilerun.png</iconset>
   </property>
   <property name="text">
    <string>编译运行(o)</string>
   </property>
  </action>
  <action name="actionSet">
   <property name="text">
    <string>设置(T)</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/zoomIn.png</normaloff>:/images/zoomIn.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/zoomOut.png</normaloff>:/images/zoomOut.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
  <include location="notepad.qrc"/>
 </resources>
 <connections/>
</ui>
